publish.py

```diff
@@ -26,9 +27,12 @@
     yaml.dump(chart, f, default_flow_style=False)
     f.truncate()
 with open('deploy/k8s/quickstart.yaml', 'w') as f:
-    subprocess.run(['helm', 'template', '--name-template', 'quickstart', 'deploy/helm/wg-access-server/'], stdout=f)
-subprocess.run(['helm', 'package', 'deploy/helm/wg-access-server/', '--destination', 'docs/charts/'])
-subprocess.run(['helm', 'repo', 'index', 'docs/', '--url', 'https://place1.github.io/wg-access-server'])
+    subprocess.run(['helm', 'template', '--name-template',
```

Comment:
I'm guessing you have a formatter in your editor that's making some of these changes. Perhaps we can add an `.editorconfig` file to avoid formatting churn. I personally prefer 120 max-line-limit rather than the 80 your formatter is using.

No worries though, i'm happy to merge anyway